### ISSUE TITLE
django version >= 2.0 removes the django.core.urlresolvers module

### ISSUE THREAD

**Issue by riberaborrell** [klen/django_markdown#81]
The django.core.urlresolvers module is removed in favor of its new location, django.urls.
https://docs.djangoproject.com/en/2.0/releases/2.0/
Therefore, such an error:
ModuleNotFoundError: No module named 'django.core.urlresolvers'

**Comment by aaamourao** [comment]
Indeed, it is preventing me to use django_markdown with my django 2.0.1 project

**Comment by Perados** [comment]
This library is not maintained anymore, any alternative to use with django 2.0+ ?

**Comment by mireille-raad** [comment]
It will be useful to add this in the README :)

**Comment by Perados** [comment]
Here is what I use now (with Django 2.0+): https://github.com/neutronX/django-markdownx

**Comment by IanTobar** [comment]
This issue isn't resolved. For anyone who wants to use django_markdown in 2.0+ you need to alter the first line in utils.py from  django.core.urlresolvers to django.urls
